packaging/nntrainer.spec

```diff
@@ -93,10 +93,12 @@ Static library package of nntrainer-devel
 Summary:	NNTrainer Examples
 Requires:	nntrainer = %{version}-%{release}
 Requires:	iniparser
+Requires:	capi-nnstreamer
```

Comment:
Shouldn't this be updated to the tizen rootstrap as well?

Reply:
this is for nntrainer applicaitions and not nntrainer/capi library.
is there any update needed in rootstrap for sample applications?

Reply:
ughh right. I thought it is nntrainer-devel because it was shrinked weiredly